lib/rules/color-function-notation/index.mjs

```diff
@@ -142,11 +144,18 @@ function isComma(node) {
 	return node.type === 'div' && node.value === ',';
 }
 
-/**
- * @param {import('postcss-value-parser').FunctionNode} node
- */
-function hasCommas(node) {
-	return node.nodes && node.nodes.some((childNode) => isComma(childNode));
+/** @param {import('postcss-value-parser').Node[]} nodes */
+function isLikelyLegacy(nodes) {
+	if (nodes.length < 5) return false;
+
+	let commas = 0;
+
+	for (const node of nodes) {
+		if (isComma(node)) commas++;
+		else if (node.value === '/') return false;
+	}
+
+	return commas && (commas === 2 || commas === 3);
```

Comment:
[suggest] `commas` (equals `commas > 0`) seems unnecessary: 👀 

```suggestion
	return (commas === 2 || commas === 3);
```

Reply:
That's an early return when you have a modern notation.
i.e. `=== 0`

Reply:
It's an early return so that you don't have to check the second condition.